packages/payload/src/admin/components/views/Global/Default.tsx

```diff
@@ -37,10 +49,25 @@ const DefaultGlobalView: React.FC<DefaultGlobalViewProps> = (props) => {
     permissions,
   } = props
 
+  const { setViewActions } = useActions()
+
   const { label } = global
 
   const hasSavePermission = permissions?.update?.permission
 
+  useEffect(() => {
+    if (hasDefaultActions(global.admin.components?.views?.Edit)) {
+      const defaultActions = global.admin.components.views.Edit.Default?.actions
```

Comment:
Again, I think this can be simplified to not need hasDefaultActions using optional chaining || [].